[bug] Fix bash execution workflow

**About Bug or problem**
Workflow for executing bash scripts not working due to permission denied error

**Steps to reproduce**
Edit a bash file in the repo and issue a pull request. The workflow check for executing the bash file fails

```
./.github/workflows/run-modified-bash.sh: line 25: conditions/if-else.sh: Permission denied
Error: Process completed with exit code 126.
```

--- a/.github/workflows/run-modified-bash.sh
+++ b/.github/workflows/run-modified-bash.sh
@@ -13,6 +13,8 @@ b=$(echo "$a" | cut -d '"' -f 2)
 b=$(echo "$b" | cut -d '[' -f 2)
 #cut ] from string
 b=$(echo "$b" | cut -d ']' -f 2)
+#remove new line character
+b=$(echo "$b" | tr -d '\n')
 
 echo "run-sh: After parsing, file to execute: $b" 
 
@@ -22,6 +24,7 @@ if [[ "$b" == *".sh"* ]]; then
 
     if [[ "$b" != *"run-modified-bash.sh"* ]]; then
         echo "run-sh: This is a .sh file, executing"
+        chmod +x "$b" # give executable permission
         $b
     else
         echo "$b was modified. No need to execute"

